routes/api.php

```diff
@@ -131,8 +131,8 @@
             'create', 'edit'
         ]);
 
-        Route::post('/places/categories/search', 'Api\Place\PlaceCategoryController@getPlaceCategoryByName');
+        Route::get('/places/categories/{id}/tags', 'Api\Place\PlaceCategoryTagsController@getTagsByCategoryId');
 
-        /* Routes here.. */
+        Route::post('/places/categories/search', 'Api\Place\PlaceCategoryController@getPlaceCategoryByName');
```

Comment:
this is `Route::get()`.

Reply:
Not mine part of code.
There is POST 'cause  this route passing data with name of the category to set it in action here -> https://github.com/BinaryStudioAcademy/bsa-2018-hedonist/blob/468ef4ffaf7bc2d63f18b75115d3fc6f0f43386b/frontend/src/store/modules/place/category/actions.js#L7